src/services/cdn.ts

```diff
@@ -1,38 +1,33 @@
 import axios from 'axios';
-import { IJobRepoLogger } from "./logger";
+import { ILogger } from "./logger";
 
 export const axiosApi = axios.create();
 
 
 export interface ICDNConnector {
     purge(jobId: String, urls: Array<string>): Promise<void>;
-    purgeAll(jobId: String, creds: any): Promise<any>;
+    purgeAll(creds: any): Promise<any>;
```

Comment:
An interface for `creds` might be useful here - it seems like we're always expecting `id` and `token` to be present?